integration-cli/docker_utils.go

```diff
@@ -86,6 +86,15 @@ func init() {
 	dockerBasePath = info.DockerRootDir
 	volumesConfigPath = filepath.Join(dockerBasePath, "volumes")
 	containerStoragePath = filepath.Join(dockerBasePath, "containers")
+	// Make sure in context of daemon, not the local platform. Note we can't
```

Comment:
I don't mind either way :) I like mine for clarity (bonus points for not having to change it), but.....